manim/mobject/coordinate_systems.py

```diff
@@ -1084,6 +1085,69 @@ def construct(self):
             **kwargs,
         ).set_opacity(opacity=opacity)
 
+    def get_area(
+        self,
+        graph: "ParametricFunction",
+        x_range: Optional[Sequence[float]] = None,
+        color: Union[Color, Iterable[Color]] = [BLUE, GREEN],
+        opacity: float = 0.3,
+        bounded: "ParametricFunction" = None,
```

Comment:
**Nitpick** : I don't really like the name `bounded`, since it often implied that the parameter is a boolean variable. Why not `bounded_func`, or something ?

Reply:
I agree, I just used the same parameter as the original one. Should I modify the name of the original one too then?

Reply:
I think you should modify it